### PR TITLE
Give some description about degenerate stadia manipulations.

### LINKED ISSUE #44: Explain degenerate stadia better
From reviewer 2:

> "Degenerate Stadia": What are the manipulations? Please detail.

### PR DESCRIPTION
**Pull request by chrisdembia**
Fixes #44.

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -406,7 +406,11 @@ \subsection*{Departures from Yeadon's work}
       denominator. In this scenario, Yeadon still employs the formulae for
       stadium solids but sets the thickness to be very
       small~\cite{Yeadon1990f}.  Instead, we manipulate the equations so that
-      the approximation is not necessary.
+      the approximation is not necessary. In the case that only one of the
+      stadia for a solid has zero thickness, the divide-by-zero issue is
+      removed by computing the mass properties for the stadium solid in which
+      the two stadia are swapped. However, this manipulation does not work if
+      both stadia have zero thickness; for this case, we refer you to the code.
```
Comment (moorepants) [inline main.tex]: I'm not sure if this will fly.

Is the code some crazy stuff? Why isn't an error thrown?

Comment (chrisdembia) [inline main.tex]: It's okay to have zero-thickness. For example, both stadia having zero thickness is a truncated cone. We should support that. So I don't think we should throw an error.

The description of the manipulations is here: https://github.com/chrisdembia/yeadon/blob/master/yeadon/solid.py#L397

I'd have to give a lot of background in the paper to actually describe the manipulation.

I'll clarify that the degeneracy is fine to have though.

File: main.tex [hunk 2]
```diff
@@ -403,10 +403,15 @@ \subsection*{Departures from Yeadon's work}
...
-      the approximation is not necessary.
+      denominator; however, the resulting shape is still valid. In this
+      scenario, Yeadon still employs the formulae for stadium solids but sets
+      the thickness to be very small~\cite{Yeadon1990f}.  Instead, we
+      manipulate the equations so that the approximation is not necessary. In
+      the case that only one of the stadia for a solid has zero thickness, the
+      divide-by-zero issue is removed by computing the mass properties for the
+      stadium solid in which the two stadia are swapped. However, this
+      manipulation does not work if both stadia have zero thickness; for this
+      case, we refer you to the code.
```
Comment (moorepants) [inline main.tex]: Should this be: "For this case we compute the inertial properties of a cylinder."

Comment (chrisdembia) [inline main.tex]: I like that!

**Comment by chrisdembia** [comment]
Updated.

**Comment by moorepants** [comment]
+1

**Pull request merged**